src/Game/AI/Action/actionCurseRRematchCount.cpp

```diff
@@ -16,4 +18,32 @@ void CurseRRematchCount::loadParams_() {
     getDynamicParam(&mGameDataStringCounterName_d, "GameDataStringCounterName");
 }
 
+bool CurseRRematchCount::oneShot_() {
+    ChampionBalladManager* manager = ChampionBalladManager::instance();
```

Comment:
```suggestion
    auto* manager = ChampionBalladManager::instance();
```

less typing and easier to read because ChampionBalladManager also appears on the right-hand side

`auto` is basically the equivalent of Java's `var`, if you're familiar with that.